pgvectorscale/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(unexpected_cfgs)]
 use pgrx::prelude::*;
```

Comment:
Not worth upgrading over, but the need for `#![allow(unexpected_cfgs)]` is fixed in future pgrx versions.